config/default/pathauto.pattern.localgov_event.yml

```diff
@@ -9,12 +9,12 @@ _core:
 id: localgov_event
 label: Event
 type: 'canonical_entities:node'
-pattern: 'events/[node:title]'
+pattern: 'children-young-people-and-families/fostering/events/[node:title]'
```

Comment:
Will this change the pattern for all events not just for fostering?

Reply:
This is a quick fix for fostering since that's the only department that has any events. We'll figure it out properly for other departments after we have this live.